src/index.ts

```diff
@@ -421,52 +419,38 @@ const main: JupyterFrontEndPlugin<IDebugger> = {
       }
     });
 
-    commands.addCommand(CommandIDs.create, {
-      label: 'Debugger',
-      execute: async () => {
-        if (sidebar) {
-          return;
-        }
+    const callstackCommands = {
+      registry: commands,
+      continue: CommandIDs.debugContinue,
+      terminate: CommandIDs.terminate,
+      next: CommandIDs.next,
+      stepIn: CommandIDs.stepIn,
+      stepOut: CommandIDs.stepOut
+    };
 
-        const callstackCommands = {
-          registry: commands,
-          continue: CommandIDs.debugContinue,
-          terminate: CommandIDs.terminate,
-          next: CommandIDs.next,
-          stepIn: CommandIDs.stepIn,
-          stepOut: CommandIDs.stepOut
-        };
-
-        sidebar = new Debugger.Sidebar({
-          service,
-          callstackCommands,
-          editorServices
-        });
-
-        sidebar.service.eventMessage.connect(_ => {
-          commands.notifyCommandChanged();
-        });
-
-        sidebar.service.sessionChanged.connect(_ => {
-          commands.notifyCommandChanged();
-        });
-
-        if (labShell.currentWidget) {
-          labShell.currentWidget.activate();
-        }
+    sidebar = new Debugger.Sidebar({
```

Comment:
One last thing: we could declare and define the `sidebar` variable here directly, with:

```typescript
const sidebar = new Debugger.Sidebar({...})
```

Reply:
rigth